### PR TITLE
Reducing length the doc while keeping the message …

### PR DESCRIPTION
**Pull request by sridharangirish91**
…consistent

### CODE REVIEW

**Comment by danwing** [comment]
github considers every line to have changed.  Here is an (easier) side-by-side diff via RFCDIFF, https://author-tools.ietf.org/api/iddiff?url_1=https://SpencerDawkins.github.io/net-collab-rqmts/draft-kwbdgrr-tsvwg-net-collab-rqmts.txt&url_2=https://SpencerDawkins.github.io/net-collab-rqmts/sridharanr/draft-kwbdgrr-tsvwg-net-collab-rqmts.txt

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md [hunk 1]
```diff
@@ -597,56 +560,40 @@ REQ-CLIENT-DECIDES is satisfied by signaling Client Flow Authorization as part o
...
+resolution information of instantaneous bandwidth available.
+
+If application packets can tolerate delay or some loss of lower priority
+packets, the network traffic shaper and scheduler can use this information
+to provide higher application quality of service. Research in {{5G-Octopus}} indicates
+that media applications can achieve better quality when sending at a higher rate
+and allowing the network to delay or drop low priority packets.
+
+The metadata in {{relative-priority}} should satisfy constraints identified
+in {{sys-considerations}}. Privacy {{privacy}} requires that metadata should not provide
```
Comment (danwing) [inline draft-kwbdgrr-tsvwg-net-collab-rqmts.md]:
```suggestion
in {{sys-considerations}}. Privacy ({{privacy}}) requires that metadata should not provide
```

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md [hunk 2]
```diff
@@ -286,44 +263,30 @@ In this document, this is termed 'Reactive Management'.
...
+{{Figure-netshaper}} shows a bottleneck (access) router on the Server to Client path.
+A network shaper in the router manages QoS of multiple users’ flows and can buffer,
+discard, or apply other flow control rules. Application layer signaling and feedback
+between Client and Server (A - in the figure) adjust transmission rate over several RTTs
+using feedback and congestion control algorithms (CCAs). CCAs are generally conservative
+and settle to a steady rate to avoid excessive packet loss. In networks where link
+conditions (between Client and Router) vary significantly at sub-RTT timescales,
+this results in unused bandwidth at short timescales.
+
+Research (e.g., {{5G-Octopus}}) indicates that media applications can achieve better QoE
```
Comment (danwing) [inline draft-kwbdgrr-tsvwg-net-collab-rqmts.md]:
```suggestion
Research (e.g., {{5G-Octopus}}) indicates that media applications can achieve better Quality of Experience (QoE)
```

IETF likes acronym expanded on first use. We don't use QoE later, but it's useful to have **QoE** in the document if someone searches for it.

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md [hunk 3]
```diff
@@ -286,44 +263,30 @@ In this document, this is termed 'Reactive Management'.
...
+packet loss or delay of low priority packets. Packet priority and tolerance to delay in
+such cases would be provided on-path in a side channel associated with the downstream packet (B).
+The requirements for this server-to-network (S2N) metadata are described in {{server-network}}.
+
+The client may provide information to an (access) router to drop lower priority marked packets
+of a flow (UDP 4-tuple) temporarily which can in turn allocate available bandwidth
+to other flows of that network attachment, especially during a reactive management event.
+
+Network shapers observe flows and apply policies to maximize performancebut are unaware underlying
+flows belinging to the same user and network attachment (e.g., a subscriber connection,a 3GPP PDU
```
Comment (danwing) [inline draft-kwbdgrr-tsvwg-net-collab-rqmts.md]:
```suggestion
flows belonging to the same user and network attachment (e.g., a subscriber connection, a 3GPP PDU
```

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md [hunk 4]
```diff
@@ -286,44 +263,30 @@ In this document, this is termed 'Reactive Management'.
...
+Session {{net-attach}}). Clients may provide information to an (access) router to drop
+‘lower priority’-marked packets of a flow (UDP 4-tuple) temporarily during congestion,
```
Comment (danwing) [inline draft-kwbdgrr-tsvwg-net-collab-rqmts.md]:
```suggestion
‘lower priority’-marked packets of a flow (UDP 4-tuple) during Reactive Management,
```

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md [hunk 5]
```diff
@@ -286,44 +263,30 @@ In this document, this is termed 'Reactive Management'.
...
+allowing bandwidth allocation to other flows of the same network attachment.
```
Comment (danwing) [inline draft-kwbdgrr-tsvwg-net-collab-rqmts.md]:
```suggestion
allowing bandwidth allocation to other packets of the same UDP 4-tuple.
```

**Review by danwing — CHANGES_REQUESTED** [review]

**Comment by boucadair** [comment]
I like the changes overall, but given that we are not sure that everyone understands what we are trying to do, I suggest we leave the elaborated version (OLD) as it providers readers with explicit details. 

We can consider merging this PR in future versions. Thanks.

**Comment by boucadair** [comment]
Thanks @sridharangirish91. You may keep the PR open, though.

**Comment by sridharangirish91** [comment]
> Thanks @sridharangirish91. You may keep the PR open, though.

Thanks Med. I'd most likely have to resolve bunch of conflicts when we decide to take it in. So I've kept the branch alive. Will resolve those after 121 and reopen the pr :)

**Pull request closed without merging**